app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+import random
+
+
+def flip_coin():
+    result_flipping = {i: 0 for i in range(11)}
+    for _ in range(10000):
+        heads = 0
+        for _ in range(10):
+            flip_result = random.randint(0, 1)
+            if flip_result:
+                heads += 1
```

Comment:
You may simplify a little, but it's up to you
```suggestion
            heads += random.randint(0, 1)
```